core/utils.ts

```diff
@@ -232,15 +233,22 @@ export interface DenoConfig {
 }
 
 export async function getDenoConfig(): Promise<DenoConfig | undefined> {
-  try {
-    const content = await Deno.readTextFile("deno.json");
-    return JSON.parse(content) as DenoConfig;
-  } catch (err) {
-    if (err instanceof Deno.errors.NotFound) {
-      return;
-    }
+  for (const configFile of ["deno.json", "deno.jsonc"]) {
+    try {
+      let content = await Deno.readTextFile(configFile);
+
+      if (configFile.endsWith(".jsonc")) {
+        content = stripJsonComments(content);
```

Comment:
I'd rather to use the Deno's standard jsonc parser library https://deno.land/std@0.151.0/encoding/jsonc.ts instead of a limited dependency from esm.sh. Note that `jsonc` is not only a json with comments, but it has additional features like not needing to quote the keys.

Reply:
🤯 I don't know there is a JSONC parser in std! thanks for the tip.

> Note that `jsonc` is not only a json with comments, but it has additional features like not needing to quote the keys.

I don't see many JSONC files in the wild not quoting the keys, but some do use trailing commas.

Reply:
Ah, yes. You're right, I thought jsonc and json5 were the same thing.